src/mmc.c

```diff
@@ -182,6 +188,8 @@ int mmc_init(unsigned int id)
 	int ret;
 
 
+	SERIAL_PUTS_ARGI("Starting init of MSC id ", id, "\n");
```

Comment:
Not really useful, you can remove this.

Reply:
I think it's usefull, since it shows from which MSC you are trying to boot